Source/Extensions/Blazorise.FluentValidation/Config.cs

```diff
@@ -0,0 +1,21 @@
+﻿#region Using directives
+using Microsoft.Extensions.DependencyInjection;
+#endregion
+
+namespace Blazorise.FluentValidation
+{
+    public static class Config
+    {
+        /// <summary>
+        /// Adds an implementation of the FluentValidationHandler implementation of the IValidationHandler interface
+        /// </summary>
+        /// <param name="services"></param>
+        /// <returns></returns>
+        public static IServiceCollection AddBlazoriseFluentValidation( this IServiceCollection services )
+        {
+            services.AddScoped<FluentValidationHandler>();
```

Comment:
Why is it not the standard, `services.AddScoped<IValidationHandler, FluentValidationHandler>();` ?

Reply:
Because we don't inject basic `IValidationHandler` anywhere. FluentValidationHandler is a special case for this extension so that's why we will inject it directly.

Reply:
I have actually started in that direction, but there was a problem. Must remember what it was.

Reply:
So, the problem with the global option is that we would not be able to distinguish between regular DataAnnotation and custom fluent validators. They both have a similar use-case API, in a way that both require a `Model` parameter. So if we use fluent validations, there would not be a way to go back to DataAnnotation. That's why `HandlerType` on a `Validations` level is better IMO, as it is more explicit. We can try to improve it in a later version if users start to request it.